### PR TITLE
feat: Add version check.

### PR DESCRIPTION
**Pull request by jasonborg**

### CODE REVIEW

File: snapshot_dbg_cli_tests/test_cli_version.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    self.urlopen_patcher = patch(
+        'snapshot_dbg_cli.http_service.urlopen', autospec=True)
+    self.urlopen_mock = self.urlopen_patcher.start()
+    self.addCleanup(self.urlopen_patcher.stop)
+
+    # Just to have a default return value set, tests that care will update it.
+    self.set_urlopen_response('foo')
+
+  def set_urlopen_response(self, body):
+    response_headers = EmailMessage()
```
Comment (mctavish) [inline snapshot_dbg_cli_tests/test_cli_version.py:45]: Why EmailMessage?

File: snapshot_dbg_cli_tests/test_init.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import unittest
+import snapshot_dbg_cli
+
+
+class CliInitTests(unittest.TestCase):
+  """ Contains the unit tests for __init__.py
+  """
+
+  def test_version_is_expected_value(self):
+    # Yes, this will need to be updated for each new version.
```
Comment (mctavish) [inline snapshot_dbg_cli_tests/test_init.py:26]: :)

**Review by mctavish — APPROVED** [review]

File: snapshot_dbg_cli_tests/test_cli_version.py (hunk 1 above)
Comment (jasonborg) [reply snapshot_dbg_cli_tests/test_cli_version.py:45]: Yes, that does look odd, but that's what is used. I've added in a comment and link to  https://docs.python.org/3/library/urllib.request.html#urllib.response.addinfourl.headers

File: snapshot_dbg_cli_tests/test_init.py (hunk 2 above)
Comment (jasonborg) [reply snapshot_dbg_cli_tests/test_init.py:26]: Not ideal, but wanted to ensure the __version__ was set to the expected version string :)

**Pull request merged**